guides/source/active_storage_overview.md

```diff
@@ -36,7 +36,8 @@ files.
 ## Setup
 
 Active Storage uses two tables in your application’s database named
-`active_storage_blobs` and `active_storage_attachments`. After creating a new
+`active_storage_blobs` and `active_storage_attachments`. If you are upgrading from an earlier version of Rails, 
+make sure you have `require "active_storage/engine` in `config/application.rb`.  After creating a new
```

Comment:
`require "active_storage/engine` does not need when required `rails/all`. 
As `README`, I think that it is good to suggest when `active_storage:install` task is not found. 
https://github.com/rails/rails/blob/master/activestorage/README.md#installation

Reply:
👍 Can you change the PR to add a NOTE like in the README?